dashboard/test/factories/factories.rb

```diff
@@ -48,7 +48,7 @@
   end
 
   factory :user do
-    birthday Date.new(1991, 3, 14)
+    birthday Time.zone.today - 21.years
```

Comment:
Note, this will fix the date when factories.rb is loaded since it's not a [dynamic attribute](https://www.rubydoc.info/gems/factory_bot/file/GETTING_STARTED.md#Dynamic_Attributes). I think that's fine, and probably better since we create a lot of test users and while minimal there's no point recalculating `today - 21.years` each time. Test runs don't last days. 👍

Reply:
TIL!